keyboards/dz60/keymaps/xtonhasvim/keymap.c

```diff
@@ -0,0 +1,94 @@
+#include "dz60.h"
```

Comment:
You can replace `#include "dz60.h" ` with `#include QMK_KEYBOARD_H`

Reply:
^

(OPTIONAL)  Also, you could probably drop the keymap into /layouts/community/60_ansi/(username)/ instead. That way, it's usable with any compatible 60% ansi board.

Reply:
I'd worry about the LED handling though. That doesn't look very standard? I'm using the capslock light to show when you're in VIM mode.

Reply:
For which part? 

The differently includes basically do the same thing, but ... we're trying to move to a more hardware agnostic approach. 

So, given the rest, leaving it here is fine, but changing the include is still recommended.

Reply:
Cool. Include is changed. I was worried about how the LED backlights are handling. In the default layout it looks like this: